news/2020-02-04-community-contributors-2019.md

```diff
@@ -0,0 +1,206 @@
+
+## Nominators
+
+In an effort to acknowledge the people who form the spine of osu!'s ranking system, we will be awarding a new form of Elite badges each year to commemorate the contributions of that year's most impressive Beatmap Nominators.
+
+<div align="center">
+    <img src="https://assets.ppy.sh/profile-badges/elite-nominator.png"/>
+</div>
+
+Only the most engaged qualify for such an accolade, those who truly stand at the top of their cohort as inarguably the most active and engaged Beatmap Nominators of all.
+
+To acknowledge these efforts, the Nomination Assessment Team have come together to write the following accolades describing what each of the Elite Nominators have done.
+
+### osu!
+
+[![](/wiki/shared/news/2020-02-04-community-contributors-2019/Mordred.jpg)](https://osu.ppy.sh/users/7265097)
+
+**Mordred** is a stand out individual who has been extremely active in all of nominating, resetting, qualifying and disqualifying maps. On top of this, his enthusiasm towards pushing full version maps has not gone unnoticed. He definitely cares quite a lot about the quality of mapping on osu!, and his persistence goes to show how he deserves the title of Elite Nominator.
+
+[![](/wiki/shared/news/2020-02-04-community-contributors-2019/Cheri.jpg)](https://osu.ppy.sh/users/5226970)
+
+**Cheri** has been a supremely active nominator and very approachable for all kinds of issues. She does not hesitate to contribute wherever she sees possible, and this also shows in her hard work checking qualified as a QAH member and actively posting problems and suggestions on maps that can be improved further! She has been excelling as a Beatmap Nominator, and her kindness makes that even better. As someone doing so well, she embodies the kind of passion needed to be an Elite Nominator.
+
+[![](/wiki/shared/news/2020-02-04-community-contributors-2019/eiri.jpg)](https://osu.ppy.sh/users/3388410)
+
+Everything you'd expect from a beatmap nominator and more, **eiri-** is definitely one of the best around. She's super active and excels in all of nominating, popping, and disqualifying maps, as well as generally checking maps in qualified. In addition to that, she's been very active in pushing maps from mappers both new and old across tons of musical genres.With everything covered, eiri- is definitely good at what she does!
+
+[![](/wiki/shared/news/2020-02-04-community-contributors-2019/bossandy.jpg)](https://osu.ppy.sh/users/360437)
+
+There are those who are incredible simply by being themselves. **bossandy** is not only a good nominator in terms of modding, but he is also very open minded about different mapping styles and always try to push maps from all kinds of difficulties regardless of how unconventional it may be and still maintain a good proficiency. His great performance paired with his very kind personality is what makes him stand out and earn not only the respect of others, but also the title of Elite Nominator!
```

Comment:
```suggestion
There are those who are incredible simply by being themselves. **bossandy** is not only a good nominator in terms of modding, but he is also very open minded about different mapping styles. He always tries to push maps from all kinds of difficulties, regardless of how unconventional it may be, while still maintain a good proficiency. His great performance paired with his very kind personality is what makes him stand out and earn not only the respect of others, but also the title of Elite Nominator!
```
The sentence in the middle is quite long, better to split it up. (By the way, not entirely sure what "proficiency" means in this context)